### PR TITLE
[CDN] Add GenericBreakingChange for all cdn command

### PR DESCRIPTION
**Pull request by ChenglongLiu**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Cdn/Cdn/Common/AzureCdnCmdletBase.cs [hunk 1]
```diff
@@ -18,12 +18,14 @@
...
 using Microsoft.Azure.Commands.ResourceManager.Common;
 using Microsoft.Azure.Management.Cdn;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
 namespace Microsoft.Azure.Commands.Cdn.Common
 {
     /// <summary>
     /// Base class of Azure Cdn Cmdlet.
     /// </summary>
+    [GenericBreakingChange("This command will be redesigned", "2.0.0")]
```
Comment (BethanyZhou) [inline src/Cdn/Cdn/Common/AzureCdnCmdletBase.cs:28]: Hi @ChenglongLiu , could you please let me what exact breaking change will be done? Here is the process for breaking change: https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/teams_docs/azps_docs/breakingchange_release_process

**Review by BethanyZhou — CHANGES_REQUESTED** [review]
To release breaking change message to customer, we need add a change log for publishing new module.

File: src/Cdn/Cdn/Common/AzureCdnCmdletBase.cs (hunk 1 above)
Comment (ChenglongLiu) [reply src/Cdn/Cdn/Common/AzureCdnCmdletBase.cs:28]: @BethanyZhou According to the meeting with powershell team, powershell based on c# SDK should be deprecated. All the CDN powershell command should generated from Swagger dicrectly, which will cause all commads different from the pervious verison.

**Comment by ChenglongLiu** [comment]
> To release breaking change message to customer, we need add a change log for publishing new module.

Added

File: src/Cdn/Cdn/ChangeLog.md [hunk 2]
```diff
@@ -20,6 +20,9 @@
 
 ## Upcoming Release
 
+## Version 1.8.2
```
Comment (BethanyZhou) [inline src/Cdn/Cdn/ChangeLog.md:23]: Please don't add version for change log.

File: src/Cdn/Cdn/Common/AzureCdnCmdletBase.cs [hunk 3]
```diff
@@ -18,12 +18,14 @@
...
 using Microsoft.Azure.Commands.ResourceManager.Common;
 using Microsoft.Azure.Management.Cdn;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
 namespace Microsoft.Azure.Commands.Cdn.Common
 {
     /// <summary>
     /// Base class of Azure Cdn Cmdlet.
     /// </summary>
+    [GenericBreakingChange("When you bump up Az.CDN module version, there will be changes to parameters and, please use the new version", "2.0.0")]
```
Comment (BethanyZhou) [inline src/Cdn/Cdn/Common/AzureCdnCmdletBase.cs:28]:
```suggestion
    [GenericBreakingChange("When you bump up Az.CDN module version, there will be changes to parameters and output, please use the new version", "2.0.0")]
```

**Review by BethanyZhou — CHANGES_REQUESTED** [review]

File: src/Cdn/Cdn/ChangeLog.md [hunk 4]
```diff
@@ -19,6 +19,7 @@
 -->
 
 ## Upcoming Release
+* Add breaking changes message for upcoming breaking change release of Version 2.0.0
```
Comment (BethanyZhou) [inline src/Cdn/Cdn/ChangeLog.md:22]:
```suggestion
* Added breaking change message for upcoming breaking change release of version 2.0.0
```

**Review by BethanyZhou — APPROVED** [review]
Looks good to me

**Pull request merged**